conf/janus.plugin.audiobridge.cfg.sample

```diff
@@ -30,6 +30,7 @@
 								; if this key is provided in the request
 ;events = no					; Whether events should be sent to event
 								; handlers (default is yes)
+use_fec = false                 ; enable or disable FEC feature (opus)
```

Comment:
Please use tabs and not spaces before the comment (as the other lines).

Reply:
done